src/libraries/Common/src/Interop/Windows/NtDll/Interop.NtStatus.cs

```diff
@@ -1,4 +1,4 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Encoding in this file was Western European and VS was complaining. All other files are UTF8, so I switched it.

Reply:
There's a weird character hidden somewhere in this file that did not fail compilation locally but is failing in the CI. I'll update the file.